src/catalog/src/durable/impls/persist.rs

```diff
@@ -73,6 +73,10 @@ pub struct PersistHandle {
     write_handle: WriteHandle<StateUpdateKind, (), Timestamp, Diff>,
     /// Read handle to persist.
     read_handle: ReadHandle<StateUpdateKind, (), Timestamp, Diff>,
+    /// Handle for connecting to persist.
+    persist_client: PersistClient,
```

Comment:
@danhhz I just want to check, is there anything scary about holding on to a `PersistClient` for the entire lifetime of the catalog?

Reply:
nope!